lib/suspenders/generators/app_generator.rb

```diff
@@ -14,6 +14,9 @@ class AppGenerator < Rails::Generators::AppGenerator
     class_option :heroku_flags, type: :string, default: "",
       desc: "Set extra Heroku flags"
 
+    class_option :ci, type: :string, default: "circle",
+      desc: "Specify a CI provider (options: circle, travis, codeship)"
```

Comment:
Looked it up. Rails uses thor, which [doesn't do validation](https://github.com/erikhuda/thor/issues/238). Do you think it would be better if the command fails immediately when the user doesn't enter a valid option, or if it just falls back to the default and provides a message like `CI provider not recognized, defaulting to CircleCI`?